test/e2e/README.md

```diff
@@ -0,0 +1,30 @@
+# E2E Test
+
+This document is to help developers understand how to run e2e test CAPDO.
+
+## Requirements
+
+In order to run the e2e tests the following requirements must be met:
+
+* Ginkgo
+* Docker
+* Kind v0.6.0+ (*Kind v0.7.0+ not working due upstream dependencies*)
```

Comment:
Should we update the documentation (getting started guide) to mention that?

Reply:
i think that is not necessary.
it's only not working when running e2e test because kind Setup() helper from capi use `kind get kubeconfig-path` that already deprecated in `v0.7.0`

when we only create cluster as usually, it's work fine